packages/docs/pages/flex.tsx

```diff
@@ -24,153 +24,227 @@ const ExampleBox: React.FC<{ vertical?: boolean }> = ({ children, vertical }) =>
 );
 
 const FlexPage = () => {
-  const items = [
-    {
-      id: 'examples',
-      title: 'Examples',
-      render: () => (
-        <>
-          <Panel>
-            <Text>A flex container used for customizable layouts.</Text>
-            <CodePreview scope={{ ExampleBox }}>
-              {/* jsx-to-string:start */}
-              <Flex
-                alignContent="stretch"
-                alignItems="stretch"
-                flexDirection="row"
-                justifyContent="flex-start"
-                flexWrap="nowrap"
-              >
-                <FlexItem alignSelf="auto" flexBasis="auto" flexGrow={0} flexOrder={0} flexShrink={1}>
-                  <ExampleBox>Item 1</ExampleBox>
-                </FlexItem>
-                <FlexItem alignSelf="auto" flexBasis="auto" flexGrow={0} flexOrder={0} flexShrink={1}>
-                  <ExampleBox>Item 2</ExampleBox>
-                </FlexItem>
-                <FlexItem alignSelf="auto" flexBasis="auto" flexGrow={0} flexOrder={0} flexShrink={1}>
-                  <ExampleBox>Item 3</ExampleBox>
-                </FlexItem>
-                <FlexItem alignSelf="auto" flexBasis="auto" flexGrow={0} flexOrder={0} flexShrink={1}>
-                  <ExampleBox>Item 4</ExampleBox>
-                </FlexItem>
-              </Flex>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Justify content">
-            <Text>
-              Flex container's can space their element with uniform spacing in-between each flex item using the{' '}
-              <Code primary>justifyContent</Code> prop.
-            </Text>
+  return (
+    <>
+      <H1>Flex</H1>
 
-            <CodePreview scope={{ ExampleBox }}>
-              {/* jsx-to-string:start */}
-              <Flex justifyContent="space-between">
-                <FlexItem>
-                  <ExampleBox>Item 1</ExampleBox>
-                </FlexItem>
-                <FlexItem>
-                  <ExampleBox>Item 2</ExampleBox>
-                </FlexItem>
-                <FlexItem>
-                  <ExampleBox>Item 3</ExampleBox>
-                </FlexItem>
-                <FlexItem>
-                  <ExampleBox>Item 4</ExampleBox>
-                </FlexItem>
-              </Flex>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Column layout">
-            <Text>
-              One way of creating a column based layout is using a combination of the <Code primary>flexWrap</Code> prop
-              on the flex container with an additional <Code primary>flexBasis</Code> prop on the flex items.
-            </Text>
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          The <Code primary>Flex</Code> component extends the <Code primary>Box</Code> utility component and provides
+          CSS flexbox container alongside flexbox properties.
```

Comment:
```suggestion
          a CSS flexbox container alongside flexbox properties.
```

Reply:
Maybe we should also link to it 🤔 
```suggestion
          <Link href="https://developer.mozilla.org/en-US/docs/Web/CSS/flex" external target="_blank">CSS flexbox</Link> container alongside flexbox properties.
```